nginx/conf.d/includes/assets.conf

```diff
@@ -1,3 +1,14 @@
+# Network storage files
+location /external {
```

Comment:
@dhruvkb @ayushjn20  Please suggest a route name for this

Reply:
'external', 'files', 'storage' are all fine names.